math/src/main/java/smile/math/matrix/SVD.java

```diff
@@ -253,7 +253,7 @@ public Cholesky CholeskyOfAtA() {
     /**
      * Solve the least squares A*x = b.
      * @param b   right hand side of linear system.
-     * @param x   the output solution vector that minimizes the L2 norm of Q*R*x - b.
+     * @param x   the output solution vector that minimizes the L2 norm of U*x - b.
```

Comment:
why U? Isn't it A?

Reply:
For DenseMatrix.svd():  

Returns the singular value decomposition. Note that the input matrix will hold U on output.

and in QR.solve() javadoc, it use QR as specific explanation instead of A

Reply:
Got it. updated.